### ISSUE TITLE
Enhancement: query also alternativeText and caption on images

### ISSUE THREAD

**Issue by catalina-cimpanu** [strapi/gatsby-source-strapi#160]
Hi!

Would it be possible to make also the alternativeText and caption be queried in graphQL from gatsby? That would be really cool.

I have it in the API
![API-image](https://user-images.githubusercontent.com/67892765/97467652-b6c89000-1944-11eb-96df-adf1109cb9e5.png)

but I can't find it in the graphQL
![graphql-image](https://user-images.githubusercontent.com/67892765/97467997-14f57300-1945-11eb-852d-38081a0f1e1e.png)

Thank you!

**Comment by gmourier** [comment]
Yes, it would be very cool to have that to avoid creating a media component made of (file, and two string field for alt and caption) as they already exists

**Comment by RobHedley** [comment]
I have exactly the same problem where content editors are adding the images and alt text etc to Strapi but we are currently having to hard code the alt text and manually change it when they do. Obviously not ideal.

Would be good to get an update on a possible ETA for this issue.

**Comment by iassainov** [comment]
Surprised that such an essential for SEO feature is not available.

**Comment by TheDevDoctor** [comment]
Has anyone found a solution to this? Still can't access the caption or alt text via graphQl?

**Comment by altonalexander** [comment]
Glad i'm not the only one looking for a solution.

**Comment by assainov** [comment]
@TheDevDoctor @altonalexander It's available in the latest version of this library. I got it working. But there are breaking changes in the way how images are queries with GraphQL.

**Comment by richardgohcr** [comment]
> @TheDevDoctor @altonalexander It's available in the latest version of this library. I got it working. But there are breaking changes in the way how images are queries with GraphQL.

@assainov What version of what library is this?

**Comment by assainov** [comment]
@richardgohcr it's `1.0.0-alpha.0`
